gulp/config.js

```diff
@@ -1,5 +1,6 @@
 var args = require('minimist')(process.argv.slice(2));
-var VERSION = args.version || require('../package.json').version;
+var version = require('../package.json').version;
+var VERSION = args.version || version;
```

Comment:
This is confusing and ambiguous. I'd prefer just inlining this.

```js
var version = args.version || require('../package.json').version;
```

Reply:
Also if you update it like that in the `const.js` file, you can just import the `VERSION` export here. This removes duplicate logic.

Reply:
that breaks the top level require only rule.

Reply:
and you can't use any of the wacy conditional require stuff with esm, so it's good to require top level imports only.

maybe use:
```js
const pkg = require('../package.json');

const VERSION = args.version || pkg.version;
```

Reply:
Yeah, the important thing is just, that the `version` and `VERSION` names are way too similar and should be somehow differentiated.